common/supplemental/supplementalData.xml

```diff
@@ -1546,7 +1546,7 @@ XXX Code for transations where no currency is involved
 		<language type="enm" scripts="Latn" alt="secondary"/>
 		<language type="eo" scripts="Latn"/>
 		<language type="es" scripts="Latn" territories="AR BO CL CO CR CU DO EA EC ES GQ GT HN IC MX NI PA PE PR PY SV UY VE"/>
-		<language type="es" territories="AD BZ DE FR GB GI PH PT RO US" alt="secondary"/>
+		<language type="es" territories="AD BZ CA DE FR GB GI PH PT RO US" alt="secondary"/>
```

Comment:
As we discussed, we need to deprecate the languageData element and restructure the tools to deprecate (and stop updating!) the data there. Remind me, do we have a ticket for doing that?

Reply:
Yes, https://unicode-org.atlassian.net/browse/CLDR-18087

Until that ticket is out of design I'll still regenerate <languageInfo> to keep it current.